variants/README.md

```diff
@@ -130,20 +135,6 @@ We use the same Rust code for all variants.
 
 ### Next Steps
 
-After the above files are in place, the variant must be added to the `variants` workspace.
```

Comment:
Why did we remove this? Is this no longer applicable?

Reply:
Correct, this is no longer applicable.